src/server/system_services/bucket_server.js

```diff
@@ -153,13 +153,20 @@ async function create_bucket(req) {
         if (req.rpc_params.namespace.write_resource && !write_resource) {
             throw new RpcError('INVALID_WRITE_RESOURCES');
         }
+        let caching = {
```

Comment:
Default should be undefined, and only set the caching config object when this bucket has enabled caching:
```suggestion
        let caching;
```

Reply:
ok